server/oauth.ts

```diff
@@ -301,8 +302,8 @@ export const performAuthorizationCodeFlow = async (
 export const verifyOAuthCookiesLocally = async (
 	req: Request,
 ): Promise<VerifiedOAuthCookies> => {
-	const accessTokenCookie = req.signedCookies['GU_ACCESS_TOKEN'];
-	const idTokenCookie = req.signedCookies['GU_ID_TOKEN'];
+	const accessTokenCookie = req.signedCookies[OAuthAccessTokenCookieName];
+	const idTokenCookie = req.signedCookies[OAuthIdTokenCookieName];
```

Comment:
this is not strictly related to my change, I just noticed we had constants defined for this

Reply:
Good spot!